core/common/src/PlanarClipMask.ts

```diff
@@ -0,0 +1,149 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module DisplayStyles
+ */
+
+import { compareNumbersOrUndefined, compareStringsOrUndefined, CompressedId64Set, Id64Set } from "@bentley/bentleyjs-core";
+
+/** The [[planarClipMask]] values that describe how planar clip masks geometry is collected
+ * @see [[PlanarClipMaskProps]]
+ * @see [[PlanarClipMaskSettings]]
+ * @beta
+ */
+export enum PlanarClipMaskMode {
```

Comment:
Can you document how each of these works? In particular, what are valid priority values, and how are they compared?

Reply:
Done.